google/cloud/testing_util/opentelemetry_matchers.h

```diff
@@ -108,6 +108,13 @@ ::testing::Matcher<SpanDataPtr> SpanHasAttributes(Args const&... matchers) {
       ::testing::IsSupersetOf({matchers...}));
 }
 
+template <typename T>
+::testing::Matcher<
+    std::pair<std::string, opentelemetry::sdk::common::OwnedAttributeValue>>
+SpanAttribute(std::string const& key) {
```

Comment:
Would it be clearer to require the user to utter the `_`?

Reply:
Yes. I wanted to do this, but gave up too early on making it work. Fixed.